xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Translations.xml

```diff
@@ -105,7 +106,17 @@ rendering.macro.microsoftStream.parameter.start.description=Start video at a spe
 rendering.macro.microsoftStream.parameter.showinfo.name=Show Info
 rendering.macro.microsoftStream.parameter.showinfo.description=Show the video info on thumbnail.
 rendering.macro.microsoftStream.parameter.autoplay.name=Auto Play
-rendering.macro.microsoftStream.parameter.autoplay.description=Whether the video should start automatically or if the user must play it.</content>
+rendering.macro.microsoftStream.parameter.autoplay.description=Whether the video should start automatically or if the user must play it.
+
+## User Profile macro
+rendering.macro.userProfile.name=User Profile
+rendering.macro.userProfile.description=Displays a user profile with custom properties
+rendering.macro.userProfile.parameter.reference.name=User
+rendering.macro.userProfile.parameter.reference.description=Reference to a user page
+rendering.macro.userProfile.parameter.properties.name=User properties
+rendering.macro.userProfile.parameter.properties.description=List of user properties to be displayed, separated by a comma
+rendering.macro.userProfile.parameter.reference.isInvalid=The User Profile macro &lt;em&gt;reference&lt;/em&gt; parameter is either missing or invalid.
```

Comment:
You shouldn't use HTML (or wiki syntax) in a translation message. People helping with translation don't now HTML most of the time and the risk or breaking it (or even worse, inserting malicious HTML) is high. If the emphasize is really important then you should use:

```suggestion
rendering.macro.userProfile.parameter.reference.isInvalid=The User Profile macro {0}reference{1} parameter is either missing or invalid.
```

and pass the HTML wrapping a translation message parameters.

Reply:
I understand, thanks for the explanation. I have replaced the emphasis by double quotes.